.github/workflows/ci.yaml

```diff
@@ -0,0 +1,183 @@
+name: CI
```

Comment:
100% pure personal preference here. Do with it as you wish:

I like to keep my tests and SA/linters in separate actions files, simply to make it obvious at a glance where a failure occurred.  And then name them accordingly.

I.e. one called 'Tests' and one called 'Linter' or 'Static Analysis'.

If we do that, I'm not sure where that leaves `todomvc_check` to go.

Reply:
That makes sense, personally I'm fine with the naming right now:
<img width="350" alt="image" src="https://github.com/gopherjs/gopherjs/assets/7207709/903e776c-1576-4864-9dbd-092ef8aa8bb5">

If we put the `Lint Checks` into it's own workflow file it would look like `Measure ...` and just not have the `CI/` prefix.

The reason I put the linter into this workflow is because it has the same `on: push:` settings (where `Measure ...` has a different one), so that all the jobs that are run together are in one workflow file (_not that important_). Also so that we could share the same `env` values. Sharing values means way we have fewer locations to update versions and experiment flags in (_doesn't happen too often to do not an issue_).

I'm cool with breaking out the Linter or leave it alone in this PR, either works for me.

Reply:
That looks fine to me.